civet.dev/cheatsheet.md

```diff
@@ -18,25 +18,114 @@ The TypeScript code on
 
 [[toc]]
 
-## Basics
+## TC39 Left a Lot on the Table
 
-### Humanize Operators
+### Default to `const` for Iteration Items
```

Comment:
Should we start with bigger features like pipes instead? Maybe it's better to ease people in, like you say, but thought I'd mention the idea.

I assume it's intentional that you have braces and parens in the for loop here, to focus on just the `const` aspect.

Reply:
Yes, I'm intentionally keeping the first examples very close to "normal" ES.

It might make sense to lead with bigger features but I think that I want the first one to be something that 95% of devs pretty much agree "yeah, that's strictly better". Even pipes was (is still) fairly contentious in the proposal.

Reply:
I prefer more basic things are listed on top.

If the Table of Contents is at the top or side, they can 'find' the bigger features readily.  & if you (@edemaine ) think Civet pipes are a 'hidden gem' make blog posts, twitter feeds & videos displaying that feature!
(I'm a big pipes proponent since Gulp)